src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -833,4 +836,11 @@ public static Entity entity( ResultSet rs, Set<UUID> authorizedPropertyTypeIds )
         return new Entity( entityKeyId, data );
     }
 
+    public static PropertySummary propertySummary( ResultSet rs ) throws SQLException {
+        UUID entityTypeID = (UUID) rs.getObject(ENTITY_TYPE_ID_FIELD);
+        UUID entitySetId = (UUID) rs.getObject( ENTITY_SET_ID_FIELD );
+        int count = rs.getInt(COUNT);
```

Comment:
This should be a ```rs.getLong(...)```  see return type for ```count(*)``` here.

Reply:
Change made